examples/vmck.nomad

```diff
@@ -0,0 +1,155 @@
+# TODO: remove hardcoded variabiles
+#       add configurationa like in https://github.com/liquidinvestigations/node/blob/master/templates/drone.nomad
+job "vmck" {
+  datacenters = ["dc1"]
+  type = "service"
+  
+  group "imghost" {
+    task "nginx" {
+      driver = "docker"
+      config {
+        image = "nginx:mainline"
+        volumes = [
+          "/usr/share/vmck-images:/usr/share/nginx/html", # TODO: add setting for host volume
+          "local/nginx.conf:/etc/nginx/nginx.conf",
+        ]
+        port_map {
+          http = 80
+        }
+        labels {
+          liquid_task = "vmck-imghost-nginx"
+        }
+      }
+      resources {
+        memory = 80
+        cpu = 200
+        network {
+          port "http" {
+            static = 10000
+          }
+        }
+      }
+      template {
+        data = <<-EOF
+          user  nginx;
+          worker_processes auto;
+
+          error_log  /var/log/nginx/error.log warn;
+          pid        /var/run/nginx.pid;
+
+          events {
+            worker_connections  1024;
+          }
+
+          http {
+            include       /etc/nginx/mime.types;
+            default_type  application/octet-stream;
+
+            log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                              '$status $body_bytes_sent "$http_referer" '
+                              '"$http_user_agent" "$http_x_forwarded_for"';
+
+            access_log  /var/log/nginx/access.log  main;
+
+            sendfile        on;
+            sendfile_max_chunk 4m;
+            aio threads;
+            keepalive_timeout  65;
+            server {
+              listen       80;
+              server_name  _;
+              access_log  /dev/stdout  main;
+              error_log /dev/stderr info;
+              location / {
+                root   /usr/share/nginx/html;
+                autoindex on;
+                proxy_max_temp_file_size 0;
```

Comment:
Does this disable temporary files?

Reply:
What I could understand from the documentation is that if proxy_max_temp_file_size is set to 0, responses from proxied servers will not be written on disk. They will only remain in a memory buffer. By default the buffer is 4kb or 8kb so it **won't** eventually eat the whole RAM. All in all this line **disables** temporary files. [Documentation](http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_max_temp_file_size)

Reply:
Perfect.